Community PR #555 (#117)
rancher/kubewarden-product-docs: docs/version-1.24/modules/en/pages/reference/threat-model.adoc

==== Mitigation

. Kubewarden provides a Software Bill Of Materials, which lists all images
  needed. This aids with Zero-Trust. The Kubernetes Administrator must verify
  the Kubewarden images, its dependencies' images, and charts out of the

>   Kubernetes cluster, in a trusted environment. You can do this with `cosign`,

Reviewer: ```suggestion
  Kubernetes cluster, in a trusted environment. 
```

>   for example. Incidentally, this is part of the implementation needed for

Reviewer: ```suggestion
You can verify this with cosign as part of the implementation needed for air-gapped installations.
```

>   air-gapped installations.

Reviewer: ```suggestion
```
